Cartfile

```diff
@@ -1 +1 @@
-github "antitypical/Result" ~> 4.0.0
\ No newline at end of file
+github "antitypical/Result" ~> 4.0.0
```

Comment:
Should we also use `4.1.0` here too?

Reply:
I think this follows the semantic versioning rules (similar to CocoaPods), which in this case specifies that it will update to all versions except any major version above 4